docs/analyzer/user_guide.md

```diff
@@ -2579,3 +2593,30 @@ void testError2() {
   int x = 1 / 0;
 }
 ```
+
+### Setting with config file
+
+Review status can be configured by a config file in YAML format. This config
+file has to represent a list of review status settings:
+
+```yaml
+- filepath: /path/to/project/test/**
```

Comment:
Can you describe the exact blob format which can be used in this YAML file?

Reply:
Done